tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -287,6 +307,19 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_sync_parser_path_arg() {
+        let input = ".sync https://example.com/api/ 316ea470-ce35-4adf-9c61-e0de6e289c59";
```

Comment:
I presume terrible things happen if we try to sync to two places at the same time, no? Follow-up issue.